splinterd/tests/framework/builders/mod.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Framework of builders for integration testing
```

Comment:
Consider renaming this module "circuit_builder". Although ScabbardCircuitBuilder isn't a CircuitBuilder, this still provides a better definition of the contents of the module.

More philosophically, is ScabbardCircuitBuilder really a ScabbardCircuitBuilderVeil? If so, this could make sense:

crate::circuit_bulder:: # pub use things from sub-modules
crate::circuit_buiider::builder # CircuitBuilder
crate::circuit_builder::error
crate::circuit_builder::veil:: # any shared veil traits, etc.
crate::circuit_builder::veil::scabbard # ScabbardCircuitBuilderVeil